### ISSUE TITLE
Question about classification accuracy

### ISSUE THREAD

**Issue by MenghaoGuo** [ranahanocka/MeshCNN#76]
Hi, @ranahanocka 

I really appreciate your work ! 
We know SHREC16 has 16 examples in the train set and 4 in the test set for each class. So it has a total of 120 examples in the test set. According to paper, meshcnn can get 98.6% accuracy on the test set and we can compute it should get (120 x 98.6%) 118.3 (not an integer) correct results which seems wrong.
The reason for this is you adopt data augmentation in test set and get more than one samples for the same example.
I want to know whether it is correct for using this data augmentation way in test set.

Thanks !

**Comment by ranahanocka** [comment]
Hi @MenghaoGuo ,

You are right -- this is a typo in the pre-print of the paper, it should have said 99.167 (119/120). In any case, the code / pre-trained weights from this repo have the correct results.

**Comment by MenghaoGuo** [comment]
Thanks for your reply !

**Comment by MenghaoGuo** [comment]
My question has been solved
